ui-pf4/src/main/webapp/src/components/add-applications-form/add-applications-serverpath-form.tsx

```diff
@@ -66,7 +66,7 @@ export const AddApplicationsServerPathForm: React.FC<AddApplicationsServerPathFo
           id="isExploded"
           name="isExploded"
           aria-label="is exploded"
-          label="Directory is an exploded Java application archive"
+          label="Directory contains the application's source code or an exploded Java application archive"
```

Comment:
Better apply the same change also to checkbox's description [`If the directory contains an exploded application, select the check box below`](https://github.com/windup/windup-web/blob/bbd5c8aa9c7a11dbdf4c4e20a8e4af5d67fd1a75/ui-pf4/src/main/webapp/src/components/add-applications-form/add-applications-serverpath-form.tsx#L58) for consistency.